stanza/text/vocab.py

```diff
@@ -134,33 +110,53 @@ def update(self, words):
         """
         return [self.add(w) for w in words]
 
-    def words2indices(self, words):
-        """
-        Convert a list of words into a list of indices.
+    def word2index(self, w):
+        return self[w]
 
-        :param words: an iterable of words to map to indices.
+    def index2word(self, i):
+        return self._index2word[i]
+
+    def freeze(self):
+        return FrozenVocab(self)
+
+    def count(self, w):
+        """Get the count for a word.
 
-        :return: the corresponding indices for each word. If a word is not found in the vocabulary then the unknown index will be returned for it.
+        :param w: a string
         """
-        return [self[w] for w in words]
+        return self._counts[w]
 
-    def indices2words(self, indices):
+    def counts_copy(self):
```

Comment:
My concern was that someone might think they can add words to the vocab by incrementing `vocab.counts`.

So what if we just let users access `_counts` if they really want it, and remove the `counts_copy` function? The underscore should be enough to imply that they shouldn't modify it.